resources/ext.wikibase.facetedsearch.js

```diff
@@ -153,35 +157,41 @@ function updateErrorState( input ) {
 }
 
 /**
- * Extracts the queries from a list facet element.
+ * Extracts the selected facet items from a list facet element.
```

Comment:
If we need to comment a function, then keeping the language consistent is important. The function refers to "facet values", whereas the comment refers to "facet items".